Resources/Prototypes/Entities/Markers/Spawners/Random/salvage.yml

```diff
@@ -40,7 +40,7 @@
       offset: 0.0
 
 - type: entity
-  name: salvage loot spawner
+  name: salvage crusher spawner
```

Comment:
Fields should be defined in this order:
type > abstract > parent > id > categories > name > suffix > description > components.